gnomad_qc/v4/create_release/create_release_sites_ht_genomes.py

```diff
@@ -1,6 +1,93 @@
 """Script to create release sites HT for v4 genomes."""
 import hail as hl
 
+# TODO: steps to create release sites HT
+# input: VRS-annotated sites HT v3.1.4
+# move SIFT and PolyPhen to insilico_predictors in vep_ht
+# remove missing fields in vep_ht
+# drop vep from site HT and merge with vep_ht
+# remove old insilico predictors and replace with new ones (the left predictors)
+# update dbSNP to dbSNP 156
+# update freq, freq_meta, freq_index_dict by integrating updates in HGDP/TGP
+# rerun inbreeding_coefficient with the callstats from the new freq
+# replace oth with remaining in global fields
+# update global fields
+
+
+def get_sift_polyphen_from_vep(vep_ht: hl.Table) -> hl.Table:
+    """
+    Get SIFT and PolyPhen scores from VEP 105 annotations.
+
+    .. note::
+     This function is to get a max of SIFT and PolyPhen scores from mane_select
+     transcripts, otherwise get a max of SIFT and PolyPhen scores from canonical
+     transcripts. It will also drop the SIFT and PolyPhen scores and predictions
+     from the transcript_consequences struct.
+
+    :param vep_ht: VEP105 annotated Hail Table.
+    :return: Hail Table with VEP105 annotations and  SIFT and PolyPhen scores
+    extracted and stored in insilico_predictors struct.
+    """
+    ht = vep_ht.select(vep_ht.vep.transcript_consequences)
+    ht = ht.explode("transcript_consequences")
+    ht = ht.select(
+        ht.transcript_consequences.mane_select,
```

Comment:
I'm reviewing code, and found this function: https://github.com/broadinstitute/gnomad_methods/blob/1bb92bb2ad69081138a5f344713ed6236de6f8aa/gnomad/utils/vep.py#L602
Is it worth changing it so we can filter mane_select too?

Reply:
Yes, good idea!

Reply:
That way, please don't start review this PR yet, once the other function is tested, I will create a PR for you to review first.

Reply:
I updated the code to use the modified function, you can start to review when you got time.